shared/src/store/action-reducers/exercise.ts

```diff
@@ -63,6 +64,10 @@ export namespace ExerciseActionReducers {
     export const pauseExercise: ActionReducer<PauseExerciseAction> = {
         action: PauseExerciseAction,
         reducer: (draftState, { timestamp }) => {
+            const lastEntry = draftState.statusHistory.at(-1);
+            if (!lastEntry || lastEntry.status === 'paused') {
```

Comment:
There is `Exercise.getStatus()`.

But this will be irrelevant when #466 is done.